### PR TITLE
chore/sc-104328/dependencies

### PR DESCRIPTION
**Pull request by busticated**
## Description

Updates dependencies, most notably major version bumps for `usb`, `@particle/device-constants`, and `webpack`.

## How to Test

1. Pull down this branch (`git pull && git checkout chore/sc-104328/dependencies`)
2. Reinstall dependencies (`npm run reinstall`)
3. Run tests (`npm run test:ci`)

**Outcome**

All tests pass both locally and in CI 👍 

## Related / Discussions

https://app.shortcut.com/particle/story/104328

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,29 +40,29 @@
...
-    "eslint": "^8.18.0",
+    "eslint": "^8.22.0",
     "eslint-config-particle": "^2.5.0",
-    "fs-extra": "^9.1.0",
+    "fs-extra": "^10.1.0",
     "mkdirp": "^1.0.4",
-    "mocha": "^7.2.0",
+    "mocha": "^9.2.2",
     "nyc": "^15.1.0",
+    "protobufjs": "^6.11.3",
```
Comment (sergeuz) [inline package.json:61]: This module can't be a dev dependency as the generated protobuf file requires it.

**Review by sergeuz — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (busticated) [reply package.json]: ah, thank you 🙏 

i added a test to help catch this sort of thing going forward in https://github.com/particle-iot/particle-usb/pull/66/commits/54211e7fc44a727953263445c82f4f78598d7b12

...then fixed the issues in https://github.com/particle-iot/particle-usb/pull/66/commits/72aa1d861a6bedac6a124dca2ef5b6e69bd78319 https://github.com/particle-iot/particle-usb/pull/66/commits/86d89a9f1c79279138f403bf2565d025149552ea

Comment (sergeuz) [reply package.json]: Awesome, thanks!

**Review by sergeuz — APPROVED** [review]

**Review by rickkas7 — APPROVED** [review]
I was able to check out a fresh copy, npm install, and run test:ci woth no errors. Tested on Mac with node 12.

**Pull request merged**